src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: $card-size;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text {
+    font-style: normal;
```

Comment:
It's default value, but it is not critical

Reply:
Default value, can be removed